tools/isobuild/compiler.js

```diff
@@ -431,7 +433,7 @@ var compileUnibuild = function (options) {
     // Find the handler for source files with this extension (unless explicitly
     // tagged as a static asset).
     let classification = null;
-    if (! fileOptions.isAsset) {
+    if ((!fileOptions.isAsset) || fileOptions.isAsset === "both") {
       classification = sourceProcessorSet.classifyFilename(
         filename, inputSourceArch.arch);
 
```

Comment:
Yeah I should comb through all of the messages and see if they still make sense. The error will definitely get triggered, but the message could be weird.